browse/factory.py

```diff
@@ -32,12 +33,15 @@ def create_web_app() -> Flask:
     Base(app)
     app.register_blueprint(ui.blueprint)
 
+    #bdc34: not sure how to add service to app context
+    app.config['listing_service'] = FakeListingFilesService()
```

Comment:
You can use the Flask ``g`` object, see http://flask.pocoo.org/docs/1.0/appcontext/#storing-data.

Reply:
I changed this to use 'g'